cmd/influxd/run/server.go

```diff
@@ -144,7 +143,6 @@ func (s *Server) Close() error {
 type Service interface {
 	Open() error
 	Close() error
-	Addr() net.Addr
```

Comment:
Turns out we don't need it.  We only need it for integration testing, and we can do type asserts in the test if we actually need it.  This makes `Services` all act/feel the same from the Server standpoint.